participant/public/locales/en/common.json

```diff
@@ -43,6 +43,15 @@
     "noAnswer": "No",
     "button": "Continue"
   },
+  "Contact": {
+    "button": "Continue",
+    "title": "Additional information",
+    "phoneNumber": "Your phone number",
+    "additionalInfo": {
+      "label": "Is there anything else you'd like to share with WIC staff?",
+      "hint": "<br>For example: Nutrition updates, household changes, or questions."
```

Comment:
I don't think we should be putting `<br>` tags into the i18n strings and we should generally avoid `<br>` tags altogether since they are display tags and we should generally use CSS to handle display.

I actually think that the [Truss `<Label>` component](https://trussworks.github.io/react-uswds/?path=/docs/components-form-elements-label--with-hint) is implemented in a way that deviates from USWDS. Though to be fair, USWDS doesn't have a separate Label component, so it's a little bit up for grabs. 

In the [USWDS file input component](https://designsystem.digital.gov/components/file-input/), they render the hint underneath the label using this kind of markup. Note that the `<label>` tag is a sibling to the `<span class="usa-hint">` tag:

```html
<div class="usa-form-group">
  <label class="usa-label">Input accepts only specific file types</label>
  <span class="usa-hint">Select PDF or TXT files</span>
  <div class="usa-file-input">...</div>
</div>
```

The [Truss Label component](https://github.com/trussworks/react-uswds/blob/main/src/components/forms/Label/Label.tsx) renders the hint span _inside_ the label tag:

```jsx
    <label
      {...labelProps}
      data-testid="label"
      className={classes}
      htmlFor={htmlFor}>
      {children}
      {hint && <span className="usa-hint">{hint}</span>}
    </label>
```

`usa-label` is set to `display: block` so I think it's fair to for us to change the `usa-hint` to be `display: block` to achieve the same behavior instead of adding `<br>` tags to our content.

Reply:
@makaelastephens I'm sorry if I made this overly complicated. I'm asking for:
- remove the `<br>` in the i18n string
- add a `display: block` to `.usa-hint`

Reply:
Taking a look

Reply:
Ugh how irritating. Ok, probably the best and fastest way to do this is:

- Remove the `display: block` from `.usa-hint` (too many other spans use this)
- On line 92 of contact.tsx, use: `hint={<div><Trans i18nKey="Contact.additionalInfo.hint" /></div>}` to make it a block element

Reply:
Hm, any idea how I can make this text normal weight? It looks like it's inheriting the font weight from the label...
<img width="1446" alt="image" src="https://user-images.githubusercontent.com/33822068/232631965-96402f5a-2371-4161-979e-cb9fb92a2a4f.png">

Reply:
I think you could do something like this:

```
.usa-label--large,
.usa-legend--large {
  @include u-text('bold');
  @include u-font-size('sans', 9); // 22px
  @include u-line-height('sans', 2); // ~25px

  .usa-hint {
  @include u-text('normal');
}
```

Reply:
And remove the stand alone:

```
.usa-hint {
    @include u-text('normal');
}
```

Reply:
This worked! Thanks